core/os/process/client.go

```diff
@@ -89,8 +131,10 @@ func Start(ctx context.Context, name string, opts StartOptions) (int, error) {
 	stdout := &portWatcher{
 		portChan: portChan,
 		stdout:   opts.Stdout,
+		portFile: opts.PortFile,
 	}
 
+	go stdout.waitForFile(ctx)
```

Comment:
Not the end of the world as gapit is the only thing that calls this, and it exits after tracing, but you could be orphaning go routines here. This could be solved with `task.WithCancel`, and defer-closing the context. That way when a port is found, the other method will give up. Just a nit though.